core/clingutils/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ foreach(dict ${stldicts})
   string(REPLACE "complex" "root_std_complex.h" header ${header})
   string(REPLACE "multi" "" header ${header})
   ROOT_GENERATE_DICTIONARY(G__std_${dict} ${header} STAGE1 MODULE ${dict}Dict LINKDEF src/${dict}Linkdef.h DEPENDENCIES  Core)
-  ROOT_LINKER_LIBRARY(${dict}Dict G__std_${dict}.cxx DEPENDENCIES  Core)
+  ROOT_LINKER_LIBRARY(std_${dict} G__std_${dict}.cxx DEPENDENCIES  Core)
```

Comment:
Is this maybe the source of https://sft.its.cern.ch/jira/browse/ROOT-8244 ? How does this renaming from `${dict}Dict` to `std_${dict}` relate to `ROOT_GENERATE_DICTIONARY`'s `MODULE ${dict}Dict`?